src/main/kotlin/blackjack/domain/BlackjackGame.kt

```diff
@@ -0,0 +1,42 @@
+package blackjack.domain
+
+class BlackjackGame(
+    userList: List<User>,
+    private val gameDeck: GameDeck = GameDeck()
+) {
+    private val users: Users
+
+    init {
+        userList.forEach { require(it.cardSize() == GAME_START_CARD_COUNT) }
+        users = Users(userList)
+    }
```

Comment:
아래와 같이 변경해도 동일하게 동작할 것 같습니다~!

가능하다면 멤버변수의 선언과 초기화가 한번에 이루어지는게 더 보기 편하다고 생각하는데 어떻게 생각하시나요?

```suggestion
    private val users: Users = Users(userList)

    init {
        userList.forEach { require(it.cardSize() == GAME_START_CARD_COUNT) }
    }
```

Reply:
유효성 검사를 먼저하려고 한 부분이었는데 말씀 하신게 더 깔끔한 것 같습니다.